Scripts/script-ExtractDomain.yml

```diff
@@ -155,8 +199,14 @@ args:
 - name: tldList
   description: Name of Demisto list to hold known TLD list, so will be ignored when
     looking for domains
+- name: extractFromEmails
+  auto: PREDEFINED
+  predefined:
```

Comment:
Need to add default values `true` to markAsIndicator and `calcReputation` - then you will not need the logic at the top of the script